ray-operator/config/samples/ray-cluster-tpu.yaml

```diff
@@ -0,0 +1,117 @@
+---
+# This template contains a Kuberay cluster using a 2x2x2 TPU v4 PodSlice.
+# To get access to TPU resources, please follow instructions in this doc:
+# https://docs.google.com/document/d/1TRwtfi2pzXbT6We0WQdwNzDdKHruk3u1XeLLJwW38kE/edit#heading=h.8ntu2hqwqvhl
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    app.kubernetes.io/name: kuberay
+    app.kubernetes.io/instance: raycluster-tpu
+  name: raycluster-tpu-kuberay
+  
+spec:
+  headGroupSpec:
+    serviceType: ClusterIP
+    rayStartParams:
+        block: "true"
+        dashboard-host: "0.0.0.0"
+        resources: "\"{\\\"google.com/tpu\\\": 4}\""
+    template:
+      spec:
+        imagePullSecrets:
+          []
+        containers:
+          - volumeMounts:
```

Comment:
Should we add `ports` info explicitly?

Reply:
Do you mean the TPU/JAX ports? I'm not sure if they are needed.

Reply:
I mean the ports for the dashboard and GCS. 

https://github.com/ray-project/kuberay/blob/4f8505560d5296127824e9e4a8765cf511a32448/ray-operator/config/samples/ray-cluster.complete.yaml#L35-L38

Reply:
Thanks, added.